protocol-list.md

```diff
@@ -1,60 +1,83 @@
-used this method in dozens of publications.  The protocol is available in
+This paper describes step-by-step, with explanatory pictures, the droplet transfer method. Yomo and collaborators have 
+used this method in dozens of publications. The protocol is available in
 [Fujii et al, 2014](https://www.nature.com/articles/nprot.2014.107).
-There is also a [JoVE article](https://www.jove.com/video/55282).
-
-## Spencer et al, JoVE, 2013
+In the following [JoVE article](https://www.jove.com/video/55282) a similar method is applied for encapsulating microspheres in liposomes. A detailed protocol is also available in the [pdf companion article](https://www.jove.com/pdf/55282/jove-protocol-55282-preparation-giant-vesicles-encapsulating-microspheres-centrifugation)
 
-The Encapsulation of Cell-free Transcription and Translation Machinery in Vesicles for the Construction of Cellular Mimics 
+### Miyazaki et al, Nature Protocols Exchange, 2015
 
-This paper presents a simple method for the preparation of vesicles, the encapsulation of transcription and translation machinery, and monitoring of protein production. A [JoVE paper and video](https://www.jove.com/video/51304/the-encapsulation-cell-free-transcription-translation-machinery) is available, based on the paper by Sunami et al., Synthesis of Functional Proteins Within Liposomes. *Methods Mol. Biol.* (2010).
+This is another description of the droplet transfer method, for the *in vitro* reconstitution of biological processes in cellular compartments. Here the [protocol](https://www.nature.com/protocolexchange/protocols/3815#/related-articles).
 
-## Ishiwata et al, Nature Protocols Exchange, 2015
-
-Preparation of cell-sized water-in-oil droplets for *in vitro* reconstitution of biological processes in cellular compartments (modified protocol)
-
-This is a [protocol](https://www.nature.com/protocolexchange/protocols/3815#/related-articles) for the encapsulation of the TX-TL cell-free protein expression system in the liposomes based on the paper by Miyazaki, M, Chiba, M, Ishiwata, S. Preparation of cell-sized water-in-oil droplets for *in vitro* reconstitution of biological processes in cellular compartments. (2015) Nature. DOI: 10.1038/protex.2015.029
-The modified version can be found at [OpenWetWare](https://openwetware.org/wiki/Preparation_of_cell-sized_water-in-oil_droplets) The protocol was used by Malin Jonsson, Miroslav Gasparek (summer 2017 interns), and Zoila Jurado (PhD student at Caltech) to express the GFP in the liposomes in the summer of 2017 in the Murray Lab at Caltech.
+A modified version of this protocol can be found at [OpenWetWare](https://openwetware.org/wiki/Preparation_of_cell-sized_water-in-oil_droplets) The protocol was used by Malin Jonsson, Miroslav Gasparek (summer 2017 interns), and Zoila Jurado (PhD student at Caltech) to express the GFP in the liposomes in the summer of 2017 in the Murray Lab at Caltech.
 
 Aaron Engelhart (University of Minnesota)
```

Comment:
We should remove this information from the protocol list.  It was put here when we were testing out GitHub at the BuildACell workshop.